include/itkNeuralNetworkFileReader.hxx

```diff
@@ -239,10 +239,10 @@ NeuralNetworkFileReader< TNetwork >
         typename tfType::Pointer tf = tfType::New();
         layerptr->SetTransferFunction(tf);
         }
-      else if ( !strcmp( (char *)mF->value, "ITK_NULLPTR" ) )
```

Comment:
This seems to be a string read from a file. Perhaps there should be no comparison here, just a plain else?

Reply:
Author: Hans Johnson <hans-johnson@uiowa.edu>
Date:   Wed Jan 10 11:10:49 2018

    BUG: Incorrect refactoring
    
    This bug was introduced in the main ITK source tree
    in:
      commit 277bf38b6d0d7cd50142f78ab4e63e8ca6bfd57a
      Author: Hans Johnson <hans-johnson@uiowa.edu>
      Date:   Fri Jan 2 08:36:11 2015
    
    Where simple search and replace was used for
    NULL keyword.

Fixed in new patch.